.travis.yml

```diff
@@ -6,52 +6,57 @@ language: rust
 services: docker
 sudo: required
 
-# This is the Rust channel that build jobs will use by default but can be
-# overridden on a case by case basis down below
-rust: 1.13.0
```

Comment:
Is there a reason not to use a default rust value?  It makes the matrix section more duplicative.